toolkit/src/styles/components/list-item.scss

```diff
@@ -0,0 +1,54 @@
+/**
+* Copyright (c) HashiCorp, Inc.
+*/
+
+@import '@hashicorp/design-system-components';
```

Comment:
should this be in the main scss file rather than the components one

Reply:
It's already imported (line 5) in the `app.scss` file so probably is a duplicate and can be removed?